proto/warden/intent/action.proto

```diff
@@ -28,9 +28,8 @@ message Action {
   // Result of the action, it will be set when the action is completed.
   google.protobuf.Any result = 6;
   string creator = 7;
-  // BTL (blocks to live) is the block height up until this action can be
-  // approved or rejected.
-  uint64 btl = 8;
+  // TimeoutHeight is the block height up until this action can be executed.
+  uint64 timeout_height = 8;
```

Comment:
The 'ActionStatus' enum in this file contains a description with the deprecated 'btl' term:

```go
enum ActionStatus {
  ...
  // Action has been rejected since Btl is expired
  ACTION_STATUS_TIMEOUT = 4;
}
```

I think it needs to be updated with new terms

Reply:
thanks for spotting it, fixed

Reply:
i'll leave spaceward for another pr, but I fixed the contracts references

openapi.yml is autogenerated but outdated so i'll leave as-is for now, we'll decide what to do with it later

Reply:
There are still some `btl` variables left in the `contracts/contracts/sample/src/contract.rs` file

Reply:
yeah you're right, sorry, i regenerates the "schema" files so now it should be done for real 😅